src/main/java/com/klodnicki/Bike/v2/DTO/bike/BikeForAdminResponseDTO.java

```diff
@@ -8,6 +8,13 @@
 
 import java.time.LocalDateTime;
 
+/**
+ * Data Transfer Object (DTO) for Bike information in the context of Admin operations. Since it is intended for Admin use, it holds
+ * more details than a typical DTO for a normal user. It is used as response.
+ * This class is annotated with @Data, @Builder, @AllArgsConstructor, and @NoArgsConstructor from the Lombok library to
+ * automatically generate boilerplate code like getters, setters, equals, hashcode, and toString methods,
+ * a builder pattern, and constructors with no arguments and all arguments.
+ */
```

Comment:
Here I'd just left two first lines. I'd build further on those instead of informing that there is `@Data` or `@Builder` used as this is not something unusual. It's fine the way you did as you explained why are those here.

Reply:
I see, I understand this. I will add then something more explanatory!